Remove preserveComments:'some' from documentation

cheshrkat opened the pull request:
As per https://github.com/gruntjs/grunt-contrib-uglify/issues/426 the option was removed, removed references to it from docs.

XhmikosR commented:
`all` doesn't exist either. Basically `preserveComments` just sets the upstream `comments` option.

avdg commented:
`all` should be replaced by `true`

cheshrkat commented:
`all` should not be replaced with true, it should map to `all` as it does in the upstream API. The way `some` worked should be mapped to `true`. See https://github.com/gruntjs/grunt-contrib-uglify/issues/430 for tests and details.

cheshrkat commented:
Updated the docs based on further discussion here and elsewhere:

- Removed all
- added an example regex to replicate what `some` used to do and to replicate the inaccessible default behaviour of Uglify's CLI API. Note this is in line with the response in the Uglify project that [people should write their own regex](https://github.com/mishoo/UglifyJS2/issues/1234#issuecomment-240730143).

cheshrkat commented:
Realised I'd edited the wrong files, will raise a fresh PR.
